submissions/IRETIOGEDENGBE/CSS/Homepage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="homepage.css" rel="stylesheet"

```

Comment:
Nice attempt. You need to check-in `homepage.css` with this PR, else the styles would not reflect